tests/e2e/src/goto.js

```
@@ -0,0 +1,23 @@
+// eslint-disable-next-line import/no-import-module-exports
+import { Config } from "../config";
+
+const pageIds = {
+  btnId: "#btn", testTypeId: "#testType", param1Id: "#param1", param2Id: "#param2", param3Id: "#param3", param4Id: "#param4",
+};
+const goto = async (currentpage, pageName) => {
+  try {
+    await currentpage.goto(`${Config.getIns().httpServer}/${pageName}`);
+  } catch (ex) {
+    // eslint-disable-next-line no-console
+    console.log(ex);
```

It might not be a good idea to hide the exception.